.cirrus.yml

```
@@ -0,0 +1,19 @@
+freebsd_task:
+  freebsd_instance:
+    matrix:
+    - image: freebsd-13-2-release-amd64
+  sysctl_script:
+    - sysctl net.inet.tcp.blackhole=0
+    - sysctl net.inet.udp.blackhole=0
+  configure_script:
+    - mkdir build
+    - cd build
+    - ../configure
+  pythoninfo_script:
+    - cd build && make pythoninfo
+  build_script:
+    - cd build && make -j$(sysctl -n hw.ncpu)
```

```suggestion
  build_script:
    - cd build && make -j$(sysctl -n hw.ncpu)
  pythoninfo_script:
    - cd build && make pythoninfo
```

Please run pythoninfo after building Python, since ``make pythoninfo``... does build Python (since it runs Python, and Python is not built yet.) :-)